app.rb

```diff
@@ -1,9 +1,24 @@
 require 'sinatra/base'
+require_relative './lib/peep_messages'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do 
+     ENV
+    @peeps = PeepMessages.all
+    erb :index
+  end 
+
+  post '/messages' do 
```

Comment:
Regarding your routes names here, you could be more explicit. Instead of **‘/messages‘** - to be more consistent with the theme of the app you could have: 

**post /peeps do**